examples/jsm/shaders/FXAAShader.d.ts

```diff
@@ -2,11 +2,11 @@ import {
   Uniform
 } from '../../../src/Three';
 
-export interface FXAAShader {
+export const FXAAShader: {
```

Comment:
All shader type declaration files use the `interface` approach. I'm afraid you can't just change a single one for consistency reasons.

What is your motivation for exporting a constant?

Reply:
Because FXAAShader is an exported variable in the js file itself, not a type.

This variable can be not a constant.
But it doesn't matter if it is a constant.

Reply:
Here is const or let or var is no longer important.

Because it is an exported object, it cannot be modified.

This also does not affect the developer to change its internal properties, such as `uniforms`.

Trust me.

Reply:
My project has not been upgraded to the latest threejs because of these errors.

Reply:
If this is really the case, we should change all respective files in `jsm/shaders`

Reply:
package.json

Reply:
What do you mean with "a lot of errors"? Please try be objective and concrete when writing such feedback.

Reply:
@FishOrBear Would be great if you can do a PR with any typescript issues you find. We do not really use typescript ourselves so we rely on typescript users to maintain them.

Reply:
I will fix the known bug, see you later

Reply:
I updated it, and now there are no compilation errors on my project.